syntax = "proto3";

// The service providing scene data.
service SceneViewer {
	// Fetch a scene.
	rpc GetScene(SceneRequest) returns (SceneResponse) {}
}

// The request message asking for the scene ID.
message SceneRequest {
	int32 sceneId = 1;
}

// The response message containing the models.
message SceneResponse {
	repeated string modelNames = 1;
}
